maintainers/maintainer-list.nix

```diff
@@ -4954,6 +4954,12 @@
     githubId = 6544084;
     name = "Kai Harries";
   };
+  kaiserkarel = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add kaiserkarel`` and sort it alphabetically.